packages/sdk/src/conversation/interface.ts

```diff
@@ -261,6 +301,64 @@ export interface TeamsFxAdaptiveCardActionHandler {
   handleActionInvoked(context: TurnContext, actionData: any): Promise<InvokeResponse>;
 }
 
+/**
+ * Interface for SSO configuration for Bot SSO
+ */
+export interface BotSsoConfig {
+  /**
+   * aad related configurations
+   */
+  aad: {
+    /**
+     * The list of scopes for which the token will have access
+     */
+    scopes: string[];
+  } & AuthenticationConfiguration;
+
+  dialog?: {
+    /**
+     * Custom sso execution activity handler class which should implement the interface {@link BotSsoExecutionActivityHandler}. If not provided, it will use {@link DefaultBotSsoExecutionActivityHandler} by default
+     */
+    CustomBotSsoExecutionActivityHandler?: new (
+      ssoConfig: BotSsoConfig
+    ) => BotSsoExecutionActivityHandler;
+
+    /**
+     * Conversation state for sso command bot, if not provided, it will use internal memory storage to create a new one.
+     */
+    conversationState?: ConversationState;
```

Comment:
Per my understanding, developers need to use non memory storage in production environment. How do we tell users about this?

Reply:
I think we can add comments in our sample or template